jest.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  roots: [
+    "<rootDir>"
```

Comment:
rootdir not defined?

Reply:
ignore this, <rootDir> apparently points to the dir that contains package.json

Reply:
Copied from SDK Node and extensions projects. Don't know what this does but will try removing it.